data/plugins.json

```diff
@@ -157,23 +206,33 @@
     "safari": 10,
     "node": 6,
     "ios": 10,
+    "chrome-flag": null,
```

Comment:
This needs to be (at least) `56` because of a bug where `return` in generators could be converted into tail calls when `--harmony` is enabled, causing nonsense crashes.